docs/DeveloperGuide.md

```diff
@@ -1103,6 +1155,35 @@ testers are expected to do more *exploratory* testing.
    3. Press `F2` or click on `Meetings` button then the `Upcoming Meetings F2` button that appears.<br>
       Expected: A `Upcoming Meetings` window appear. The meeting added in step 3 is displayed on the list.
 
+### Update client information:
+
+1. Updating client information though input shortcut:
+   
+   1. Test case: `u 1 nw/$22000`<br>
+      Expected: Networth of first client change to `$22000`. New client detail shown in command display.
+
+### Update descriptions:
+
+1. Adding a description
+
+   2. Test case: `description 2 d/Eager to invest`<br>
+      Expected: Description of the second client changed to `Eager to invest`. 
+   
+   3. Test case: `description d/Eager to invest`<br>
+      Expected: No change in descriptions made. Error details shown in command display.
+  
+   3. Other test cases: `description`, `description 0 d/Eager to invest`<br>
+      Expected: No change in descriptions made. Error details shown in command display.
```

Comment:
The prefix should be ds/ instead of d/

Reply:
yep. was running the old version. thanks!